src/wallet/mod.rs

```diff
@@ -438,11 +438,15 @@ where
         self.database.borrow().get_tx(txid, include_raw)
     }
 
-    /// Return the list of transactions made and received by the wallet
+    /// Return an unsorted list of transactions made and received by the wallet
     ///
     /// Optionally fill the [`TransactionDetails::transaction`] field with the raw transaction if
     /// `include_raw` is `true`.
     ///
+    /// To sort transactions, the following code can be used:
+    /// tx_list.sort_by(|a, b| b.confirmation_time.as_ref().unwrap().height.cmp(
+    ///     &a.confirmation_time.as_ref().unwrap().height));
+    ///
```

Comment:
Since it's going into the doc, it's better to put this code inside ```, which changes the code from this:
![Selection_087](https://user-images.githubusercontent.com/25042473/181505448-69c2615d-a2b1-4a21-b5e5-fbdd43600933.png)

to this:
![Selection_088](https://user-images.githubusercontent.com/25042473/181505440-2a3a9134-f5c6-4684-b1ee-238d70d5a784.png)

Then, you should run `cargo fmt --all -- --config format_code_in_doc_comments=true` to reformat the code.

Also, I don't really like the fact that this code panics if `confirmation_time` is `None`... Instead, we should use `map`:
```
                tx_list.sort_by(|a, b| {
                    b.confirmation_time
                        .as_ref()
                        .map(|t| t.height)
                        .cmp(&a.confirmation_time.as_ref().map(|t| t.height))
                });
```

Reply:
Done in https://github.com/bitcoindevkit/bdk/pull/686/commits/a35662cc6a908b68d5e46cccd361fa373dbabbf4. Thanks for review.